lib/internal/blocklist.js

```diff
@@ -43,6 +43,15 @@ class BlockList {
     this[kHandle][owner_symbol] = this;
   }
 
+  /**
+   * Returns true if the value is a BlockList
+   * @param {any} value
+   * @returns {boolean}
```

Comment:
I think we could use `asserts` here
```suggestion
   * @returns {asserts value is BlockList}
```

Reply:
I'm not quite sure what this would accomplish. The method does not assert at all, it just returns true or false and should not throw any errors.